framework/src/wifi_manager/wifi_manager_dhcps.c

```diff
@@ -20,12 +20,12 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include <string.h>
-
 #include <sys/types.h>
 #include <netutils/netlib.h>
+#include <tinyara/net/netlog.h>
 #include "wifi_manager_error.h"
 #include "wifi_manager_dhcp.h"
-#include "wifi_manager_log.h"
+#define TAG "[WM]"
```

Comment:
How about changing [DHCPS]?

Reply:
it's wifi_manager module so I think wm is better.